src/components/manage/Blocks/Listing/QuerystringWidget.jsx

```diff
@@ -307,7 +307,7 @@ class QuerystringWidget extends Component {
                           components={{ DropdownIndicator, Option }}
                           value={{
                             value: row.i,
-                            label: indexes[row.i].title,
+                            label: indexes[row.i]?.title,
```

Comment:
@iFlameing did you try what happens when value label has when indexes[row.i] is Null (or something else)?